src/rating/rating.jsx

```diff
@@ -433,9 +433,12 @@ export default class Rating extends Component {
                     >
                         <form action="#">
                             {enableA11y ? (
-                                <div role="group" aria-labelledby={id}>
-                                    {' '}
-                                    overlay{' '}
+                                <div
+                                    role="group"
+                                    aria-labelledby={id}
+                                    aria-label="Rating Options"
```

Comment:
@mark-ck the `aria-label` should be attached to wherever the `aria-labelledby` points to. And "Rating Options` is in english so we should add i18n support

Reply:
I have noticed that😂.
I tried that the `aria-label` won't be read by `VoiceOver`, so i didn't add i18n support.
But you are right, i'll add i18n support later.